contributors/devel/sig-instrumentation/logging.md

```diff
@@ -4,14 +4,14 @@ The following conventions for the klog levels to use.
 [klog](http://godoc.org/github.com/kubernetes/klog) is globally preferred to
 [log](http://golang.org/pkg/log/) for better runtime control.
 
-Shared libraries, such as `client-go`, should not use `klog.Errorf()` and `klog.Warningf()`,
+Shared libraries, such as `client-go`, should not use `klog.ErrorS()` and `klog.InfoS()`,
 but just return `error`, because client libraries may be used in CLI UIs that wish to control output.
 
-* klog.Errorf() - Always an error
+* klog.ErrorS() - Always an error
```

Comment:
Let's update it based on https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#using-errors

`Errors should be used to indicate unexpected behaviours in code, like unexpected errors returned by subroutine function calls`